python/cuml/test/test_random_forest.py

```diff
@@ -123,7 +123,7 @@ def test_rf_regression(datatype, split_algo, mode,
 
     # Initialize and fit using cuML's random forest regression model
     cuml_model = curfr(max_features=max_features, rows_sample=rows_sample,
-                       n_bins=16, split_algo=split_algo, split_criterion=2,
+                       n_bins=64, split_algo=split_algo, split_criterion=2,
```

Comment:
are we able to run the quality and stress tests with the large `n_bins` value?

Reply:
Oops, sorry this was a stray change from another test. I will remove it.